packages/apps/tools/src/utils/persist.ts

```diff
@@ -9,39 +9,10 @@ export const encode = (value: unknown) =>
 
 export const parse = (value: string) => JSON.parse(decodeURIComponent(value));
 
-export const setItem = (key: string, value: unknown) => {
+export const setItem = (key: string, value: unknown) =>
   Cookies.set(getName(key), encode(value));
-};
 
 export const getItem = (key: string) => {
-  try {
-    const cookie = Cookies.get(getName(key));
-    return cookie !== undefined ? parse(cookie) : undefined;
-  } catch (e) {
-    return undefined;
-  }
-};
-
-export const deleteItem = (key: string) => {
```

Comment:
Out of curiosity; Why are these deleted?

Reply:
They were not used anymore

Reply:
Were they ever used? I think in this case these could stay in, as they could potentially be used later. What do you think?

Reply:
Not a problem to retrieve them

Reply:
If you can run the formatter then that would be great, I'm approving in the meantime 😄